package.json

```diff
@@ -91,6 +94,7 @@
 		"tree-sitter-ruby": "github:tree-sitter/tree-sitter-ruby#1ebfdb288842dae5a9233e2509a135949023dd82",
 		"tree-sitter-rust": "^0.19.0",
 		"tree-sitter-scala": "github:tree-sitter/tree-sitter-scala#master",
+		"tree-sitter-scss": "github:serenadeai/tree-sitter-scss#master",
```

Comment:
Best to pin to a specific commit rather than using `master`, because tree-sitter grammars aren't tested against web-tree-sitter, and we don't have tests here, so things could break at any time 😕. I'd just look up the current tip of master and use that here

Reply:
Yep, sounds good. Should we do this for the other languages referencing a branch instead of a version/sha as well?

Reply:
Yeah we should prob pin the others too, but we'll want to do a local test to make sure we don't break cursorless before we publish that